src/pages/cluster/ClusterGroupForm.tsx

```diff
@@ -85,7 +86,17 @@ const ClusterGroupForm: FC<Props> = ({ group }) => {
         .then(() => {
           const verb = group ? "saved" : "created";
           navigate(`/ui/cluster/group/${values.name}`);
-          toastNotify.success(`Cluster group ${values.name} ${verb}.`);
+          toastNotify.success(
+            <>
+              Cluster group{" "}
+              <ResourceLink
+                type="cluster-member"
+                value={values.name}
+                to={`/ui/cluster/group/${values.name}`}
+              />{" "}
+              {verb}.
+            </>,
+          );
```

Comment:
I think we should always keep the resource type as part of the copy, like in this example. That makes it easy to learn the icons and reduces the cognitive load, i.e. when browsing notifications. Currently, you'd need to understand what each icon stands for to process a notification. Putting the type name as an explicit part of the copy makes it much easier to process.

Reply:
This is another design decision we need to align on with @piperdeck. I believe the purpose of using icons is to eliminate the need for resource type text in notifications. In last week’s discussion, we agreed that only storage pools and custom volumes should include their types in the notification copy, as we don’t yet have representative icons for them. Before proceeding with this change, I’d like to confirm this approach, as it will require updates to all the e2e tests.

Reply:
Sure, lets discuss with piper. When I was playing with this branch I found it hard to digest the notifications, hence my suggestion.

> Before proceeding with this change, I’d like to confirm this approach, as it will require updates to all the e2e tests.

I thought this would actually make the e2e test easier, because we had this in the past. So with this suggestion, the CI suite would need fewer adjustments.

Reply:
Aaah right I already made adjustments to all the e2e tests :sweat_smile: but it's easy enough to reset the commit and remove those changes, but let's first confirm the change with piper. I do agree that it's additional learning if we remove the resource type copy text from the notifications

Reply:
@edlerd after discussing with @piperdeck I've added the resource type back into all notifications. Also reverted all the e2e test changes.

Reply:
Related to this [comment](https://github.com/canonical/lxd-ui/pull/952#issuecomment-2414631370) from #952 . I have adjusted the instance side navigation tab and empty state to use the `pod` icon.